snap/hooks/prepare-device

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -eux
+
+exec >> $SNAP_COMMON/prepare-device-hook.log 2>&1
+
+# TODO: define your model's API key here
+# See https://ubuntu.com/core/services/guide/serial-vault-overview
+# for instructions on how to generate an API key
+# DO NOT check this API key into a publicly accessible VCS
+MODEL_APIKEY=""
```

Comment:
Considering this line simply replaces `MODEL_APIKEY` with an empty string, we should probably use something like `${MODEL_APIKEY:-""}`

Reply:
This runs as a hook, so where else would MODEL_APIKEY get set?

Reply:
Oh I guess you could set as an env var in snapcraft.yaml

Reply:
Fair enough